From bc966656d8598a6d87c901a67ecdb91c436f4939 Mon Sep 17 00:00:00 2001
From: Herve Codina <herve.codina@bootlin.com>
Date: Wed, 29 Sep 2021 15:08:33 +0200
Subject: [PATCH] build: Use MYSQL_CONFIG variable instead of
 'mysql_config'

The MYSQL_CONFIG variable is used to check 'mysql_config' prog.
This variable can be override by the user but was not used for
retrieving MYSQL_INCLUDE, MYSQL_LIBS, ...

This commit replaces the hardcoded usage of 'mysql_config' by
the use of MYSQL_CONFIG variable.

This lead to 'MYSQL_CONFIG=/some/where/mysql_config ./configure'
command to work properly.

Signed-off-by: Herve Codina <herve.codina@bootlin.com>
---
 acinclude.m4 | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/acinclude.m4 b/acinclude.m4
index e99318f..9dd2b0f 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -116,15 +116,15 @@ if test "$ac_mysql" = "yes"; then
 	fi
 
 	if test "$ac_mysql_incdir" = "no"; then
-		MYSQL_INCLUDE=`mysql_config --include`
+		MYSQL_INCLUDE=`$MYSQL_CONFIG --include`
 	else
 		MYSQL_INCLUDE=-I$ac_mysql_incdir
 	fi
 	if test "$ac_mysql_libdir" = "no"; then
 	   	if test "$ac_mysql_threadsafe" = "YES"; then
-			MYSQL_LIBS=`mysql_config --libs_r`
+			MYSQL_LIBS=`$MYSQL_CONFIG --libs_r`
 		else
-			MYSQL_LIBS=`mysql_config --libs`
+			MYSQL_LIBS=`$MYSQL_CONFIG --libs`
 		fi
 	else
 	   	if test "$ac_mysql_threadsafe" = "YES"; then
-- 
2.31.1

